src/classes/project_data.py

```diff
@@ -947,7 +942,8 @@ def check_if_paths_are_valid(self):
                 if path and is_modified and not is_skipped:
                     # Found file, update path
                     file["path"] = path
-                    get_app().updates.update_untracked(["import_path"], os.path.dirname(path))
+                    settings
```

Comment:
What is this?

Reply:
:grimacing: an unfinished change. It was going to be `settings.setDefaultPath...` fixing